packages/plugin-team/test/unit/spec/team.js

```diff
@@ -0,0 +1,47 @@
+/**!
+ *
+ * Copright(c) 2015 Cisco Systems, Inc. See LICENSE file.
+ * @private
+ */
+
+import {assert} from '@ciscospark/test-helper-chai';
+import Team from '../..';
+import MockSpark from '@ciscospark/test-helper-mock-spark';
+
+describe(`plugin-team`, () => {
+  describe(`Team`, () => {
+    let spark;
+
+    beforeEach(() => {
+      spark = new MockSpark({
+        children: {
+          team: Team
+        }
+      });
+    });
+
+    describe(`#create()`, () => {
```

Comment:
Can you follow the same standard as previous test for 
() => assert.isRejected(); );